rust/lang/security/audit/dangerous-exec-command.yaml

```diff
@@ -0,0 +1,32 @@
+rules:
+  - id: dangerous-exec-command
+    patterns:
+      - pattern-either:
+          - patterns:
+              - pattern-either:
+                  - pattern: |
+                      Command::new($CMD)
+          - patterns:
+              - pattern-either:
+                  - pattern: |
+                      Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh|ls)/").arg("-c").arg($CMD)
+                  - pattern: |
+                      Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh)/").arg("-c").arg($CMD).output()
+                  - pattern: |
+                      Command::new("=~/(sh|bash|ksh|csh|tcsh|zsh)/").arg("-c").arg($CMD).output().expect(...)
+      - pattern-inside: |
+          use std::process::Command;
+          ...
+      - pattern-not-inside: |
+          $CMD = "...";
+          ...
```

Comment:
Can you add testcases for this? It doesn't seem to be working correctly. It seems like with this you want to exclude harcoded strings, but the testcases in the testcode all have hardcoded strings.

I want to see some `ok: dangerous-exec-command` testcases. This rule is likely to be too noisy in its current form.